apps/taxes/src/modules/avatax/maps/avatax-calculate-taxes-map.ts

```diff
@@ -0,0 +1,105 @@
+import { LineItemModel } from "avatax/lib/models/LineItemModel";
+import { TransactionModel } from "avatax/lib/models/TransactionModel";
+import { TaxBaseFragment } from "../../../../generated/graphql";
+
+import { DocumentType } from "avatax/lib/enums/DocumentType";
+import { ChannelConfig } from "../../channels-configuration/channels-config";
+import { taxLineResolver } from "../../taxes/tax-line-resolver";
+import { CalculateTaxesResponse } from "../../taxes/tax-provider-webhook";
+import { CreateTransactionArgs } from "../avatax-client";
+import { AvataxConfig } from "../avatax-config";
+import { avataxAddressFactory } from "./address-map";
+import { numbers } from "../../taxes/numbers";
+
+// * Shipping is a regular line item in Avatax
+// https://developer.avalara.com/avatax/dev-guide/shipping-and-handling/taxability-of-shipping-charges/
+export const SHIPPING_ITEM_CODE = "Shipping";
+
+function mapLines(taxBase: TaxBaseFragment): LineItemModel[] {
+  const productLines = taxBase.lines.map((line) => ({
+    amount: line.unitPrice.amount,
+    taxIncluded: line.chargeTaxes,
+    // todo: get from tax code matcher
+    taxCode: taxLineResolver.getLineTaxCode(line),
+    quantity: line.quantity,
+  }));
+
+  if (taxBase.shippingPrice.amount !== 0) {
+    // * In Avatax, shipping is a regular line
+    const shippingLine: LineItemModel = {
+      amount: taxBase.shippingPrice.amount,
+      itemCode: SHIPPING_ITEM_CODE,
+      // todo: add taxCode
+      // * Different shipping methods can have different tax codes
+      // https://developer.avalara.com/ecommerce-integration-guide/sales-tax-badge/designing/non-standard-items/\
```

Comment:
nitpick, but can you use multiline comments? `/** */`? Its better handler by IDEs

Reply:
https://eslint.org/docs/latest/rules/multiline-comment-style

Reply:
Yes, it works with two asterisks, but no eslint rule option uses two asterisks. I can add it anyway and will add the asterisk manually when it's needed.